ivy/functional/frontends/torch/Utilities.py

```diff
@@ -0,0 +1,10 @@
+#local
```

Comment:
File names should be all lowercase, matching the format of the rest of the codebase

Reply:
having problem changing file name it is showing error in vs code

Reply:
The file name here still needs to be changed. it should be `ivy/functional/frontends/torch/utilities.py`, not `ivy/functional/frontends/torch/Utilities.py`

Reply:
You've made copies of the files with lowercase. Thats fine, but can you also delete the old files with uppercase?